File: Strivers A2Z DSA Sheet/Patterns Printing/Pattern01.java
import java.util.*;

class PatternPrint {
    private int iSize;

    public PatternPrint(int iNo) {
        this.iSize = iNo;
    }

    public void PatternNxN() {
        for (int i = 0; i < iSize; i++) {
            for (int j = 0; j < iSize; j++) {
                System.out.print(" * ");
            }
            System.out.println();
        }
    }
}

class Pattern01 {
    public static void main(String Args[]) {
        int iValue = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the Pattern Size : ");
        iValue = sobj.nextInt();

        PatternPrint pobj = new PatternPrint(iValue);
        pobj.PatternNxN();

        sobj.close();

    }
}

File: Strivers A2Z DSA Sheet/Patterns Printing/Pattern01.cpp
#include <iostream>
using namespace std;

class Pattern01
{
private:
    int iNO;

public:
    Pattern01(int iSize)
    {
        iNO = iSize;
    }

    void PatternNxN()
    {
        for (int i = 0; i < iNO; i++)
        {
            for (int j = 0; j < iNO; j++)
            {
                cout << " *  ";
            }
            cout << endl;
        }
    }
};

int main()
{
    int iValue = 0;
    cout << "Enter the Number : " << endl;
    cin >> iValue;

    Pattern01 *pobj = new Pattern01(iValue);
    pobj->PatternNxN();
    delete(pobj);

    return 0;
}